packages/host/app/services/card-service.ts

```diff
@@ -87,13 +87,11 @@ export default class CardService extends Service {
   async fetchJSON(
     url: string | URL,
     args?: RequestInit,
-    loader?: Loader,
```

Comment:
same comment--lets try to limit our reliance on the loader service--if the caller passes in a loader, we should use it